aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan

```diff
@@ -107,10 +107,21 @@ type opennebula_ignoremac = {
     "interface" ? valid_interface_ignoremac[]
 };
 
+@documentation{
+Type that changes resources owner/group permissions.
+By default opennebula-aii generates all the resources as oneadmin owner/group.
+}
+type opennebula_permissions = {
+    "owner"  ? string # id or user name
```

Comment:
add the comment in an annotation `@{}`, and clarify the the `id` here is the (internal) ONE id of the user.